bot/database/init_db.sql

```diff
@@ -23,22 +24,19 @@ CREATE INDEX IF NOT EXISTS idx_user_roles_admin ON user_roles (is_admin);
 CREATE INDEX IF NOT EXISTS idx_user_roles_moderator ON user_roles (is_moderator);
 
 CREATE TABLE IF NOT EXISTS user_logs (
-    id SERIAL PRIMARY KEY,
```

Comment:
idk man ale jak chcesz zmieniac bazke to powinienes docelowo potem pisac transformacje na dole, no chyba ze wywalasz baze za kazdym razem xd

Reply:
XD to ten cooldown zem robił zaraz po reworku bazy i skumałem wtedy że się to jebie i jedna rzecz jest źle zrobiona XD

Reply:
A dużo logów żem nie stracił przez te kilka godzin tylko mojego devowania XD to tak wyszło